account_banking_pain_base/models/account_payment_order.py

```diff
@@ -706,3 +706,19 @@ def generate_creditor_scheme_identification(
         csi_scheme_name_proprietary = etree.SubElement(csi_scheme_name, "Prtry")
         csi_scheme_name_proprietary.text = scheme_name_proprietary
         return True
+
+    @api.onchange("payment_mode_id")
+    def onchange_payment_mode(self):
```

Comment:
Why not doing `batch_booking` a computed writable field now that this tool is available?